src/payment/card/stripe.js

```diff
@@ -119,7 +120,23 @@ export default class StripeCardPayment extends Payment {
 
     await this.loadScripts(this.scripts);
 
-    return this._confirmCardPayment(intent);
+    try {
+      const result = await this._confirmCardPayment(intent);
+
+      return result;
+    } finally {
+      await this._resetAsyncPayment(payment.id);
```

Comment:
Should we only reset payment status on error?

Reply:
No, if we want to get the transaction state as quickly as possible, then we should reset the asynchronous payment state.

For example, if a payment has been pending for a long time, then we will need to wait some time to receive an updated transaction status.

If we reset the asynchronous payment state, we will get a new transaction state after 30 seconds.